nautobot/extras/querysets.py

```diff
@@ -224,6 +224,15 @@ class JobQuerySet(RestrictedQuerySet):
     Extend the standard queryset with a get_for_class_path method.
     """
 
+    def delete(self):
```

Comment:
Not entirely sure we want to prevent deletes at the queryset/ORM level - will this cause us problems in the future if we have a migration that renames or obsoletes an existing system job? Not sure.

If we are going to keep this, let's make sure we have some test coverage for this case.

Reply:
The reason why I overrode this is that we are using qs.delete() in our BulkDeleteView. https://github.com/nautobot/nautobot/blob/develop/nautobot/extras/utils.py#L716